app/screens/Chat/components/RenderMessageImage/RenderMessageImageStyles.ts

```diff
@@ -14,9 +15,14 @@ export const styles = StyleSheet.create<Styles>({
     borderRadius: 20,
     marginBottom: 5,
     overflow: 'hidden',
+    marginHorizontal: 2,
   },
 
   wrapperImageStyle: {
     backgroundColor: palette.white,
   },
+
+  listImages: {
+    backgroundColor: palette.white,
```

Comment:
Do we really need this, otherwise, please try to make it better view

Reply:
will do in other PR